modules/transformer-batch/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/batch/Transformer.scala

```diff
@@ -107,7 +107,10 @@ object Transformer {
       Transformed.Shredded.Json(false, vendor, name, model, data)
     }
 
-    def typesInfo: TypesInfo = TypesInfo.Shredded(typesAccumulator.value.toList)
+    def typesInfo(skipSchemas: List[SchemaCriterion]): TypesInfo = {
```

Comment:
At the moment, `skipSchemas` is only used to remove respective schema items from shredding complete message. It isn't used to skip the respective entities during the transformation. Let me know if you think we should skip them in the transformed data as well.

Reply:
Good points, Ian! I just tested the scenario you described for Parquet and indeed we get error in that case. I guess it can be quickly solved with adding filter in [here](https://github.com/snowplow/snowplow-rdb-loader/blob/741f7f114ad0ff1a62bd5b70068a76c65024eba4/modules/transformer-batch/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/batch/ShredJob.scala#L234).

JSON transformation is more tricky because `Event.toJson` function from AnalyticsSDK. We need to make necessary changes in there.

Reply:
Let's fix the parquet scenario, since you say it's relatively easy and since you've confirmed it breaks in an ugly way.

Let's not change the analytics sdk.  This feels like too much effort to justify the benefit.

Do you agree that's the right compromise?

Reply:
Yep, sounds good to me.

Reply:
I've made the change. Could you take a look at it, @istreeter ?